docusaurus/docs/learn/core-concepts/services/overview.mdx

```diff
@@ -143,3 +143,339 @@ This strategy drives costs in the same way as the `smartrouting` strategy. Howev
+```
+
+The default cost and precedence for an identity can also be set.
+
+```
+ziti edge update identity edge-router-1 --default-hosting-precedence required --default-hosting-cost 100
+```
+
+#### Multiple Application Endpoints
+Next, let us add a second application endpoint. We want traffic load-balanced across the endpoints equally. We're going
+to do this by adding the second endpoint to the configuration.
+
+```
+ziti edge update config test-host-config host.v2 --data '
+{
+    "terminators" : [
+        {
+            "address": "192.168.3.136",
+            "port" : 8080,
+            "protocol": "tcp"
+        },
+        {
+            "address": "192.168.3.137",
+            "port" : 8080,
+            "protocol": "tcp"
+        }
+    ]
+}
+'
+```
+
+Now each ER/T will create two terminators, one for each endpoint, for a total of four terminators. Now that we have
+multiple endpoints we'll want to know when they are healthy or unavailable we can use the just the endpoints which
+are working. We can accomplish this by adding health checks to the configuration.
+
+```
+ziti edge update config test-host-config host.v2 --data '
+{
+    "terminators" : [
+        {
+            "address": "192.168.3.136",
+            "port" : 8080,
+            "protocol": "tcp",
+            "portChecks" : [
+                {
+                    "address" : "192.168.3.136:8080",
+                    "interval" : "5s",
+                    "timeout" : "100ms",
+                    "actions" : [
+                        {
+                            "trigger" : "fail",
+                            "consecutiveEvents" : 3,
+                            "action" : "mark unhealthy"
+                        },
+                        {
+                            "trigger" : "pass",
+                            "consecutiveEvents" : 3,
+                            "action" : "mark healthy"
+                        }
+                ]
+            }
+            ]
+        },
+        {
+            "address": "192.168.3.137",
+            "port" : 8080,
+            "protocol": "tcp",
+            "portChecks" : [
+                {
+                    "address" : "192.168.3.137:8080",
+                    "interval" : "5s",
+                    "timeout" : "100ms",
+                    "actions" : [
+                        {
+                            "trigger" : "fail",
+                            "consecutiveEvents" : 3,
+                            "action" : "mark unhealthy"
+                        },
+                        {
+                            "trigger" : "pass",
+                            "consecutiveEvents" : 3,
+                            "action" : "mark healthy"
+                        }
+                    ]
+                }
+            ]
+        }
+    ]
+}
+'
+```
+
+Our configuration has gotten quite large! However, we've gained a good bit of functionality with our new additions.
```

Comment:
>Our configuration has gotten quite large! However, we've gained a good bit of functionality with our new additions.

good for a blog post/guide/walkthrough, but not for a 'core concept' type of doc. Maybe all this would be better placed as a "how to health check" guide?